core/arch/arm/kernel/secure_partition.c

```diff
@@ -352,6 +382,30 @@ static TEE_Result sp_dt_get_u32(const void *fdt, int node, const char *property,
 	return TEE_SUCCESS;
 }
 
+static TEE_Result sp_dt_get_u16(const void *fdt, int node, const char *property,
+				uint16_t *value)
+{
+	const fdt16_t *p = NULL;
+	uint32_t v = 0;
+	int len = 0;
+
+	p = fdt_getprop(fdt, node, property, &len);
+	if (!p)
+		return TEE_ERROR_ITEM_NOT_FOUND;
+
+	if (len != sizeof(*p))
+		return TEE_ERROR_BAD_FORMAT;
+
+	v = fdt16_to_cpu(*p);
+
+	if (v > UINT16_MAX)
```

Comment:
This can't happen, you can assign `*value` directly and return OK.

Reply:
You are right. Corrected